### DISCUSSION TITLE
Is there an easy way to change/customize the arrows ?

### DISCUSSION THREAD

**Discussion by Bokomoko** [yifaneye/react-gallery-carousel#10] [Q&A] [answered]
How to resize, change color or even the symbol of the arrows in the carousel ?

**Comment by yifaneye** [comment, marked as answer]
Hi @Bokomoko ! Thank you for raising this question up! 😀 To change the symbol of the arrows in the carousel, use the **`leftIcon`** prop and the **`rightIcon`** prop. Unfortunately, there is currently no way to resize, change color of the default arrows in the carousel. However you can **customize the icon from my implementation** at https://github.com/yifaneye/react-gallery-carousel/blob/master/src/components/IconButton/IconButton.js#L8 and put your customised versions into the **`leftIcon`** prop and the **`rightIcon`** prop respectively. 

Hope this helps!
